react-16-17-18-ssr-with-adapters/package.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Get an error during `yarn install` Error: Couldn't find package "react-version-adapter@0.1.0" required by "@react-ssr/remote2@0.0.0" on the "npm" registry.

Reply:
Fixed :rocket: